contributors/devel/cri-validation.md

```diff
@@ -0,0 +1,48 @@
+# Container Runtime Interface (CRI) Validation Testing
+
+CRI validation testing provides a test framework and a suite of tests to validate that the Container Runtime Interface (CRI) server implementation meets all the requirements. This allows the CRI runtime developers to verify that their runtime conforms to CRI, without needing to set up Kubernetes components or run Kubernetes end-to-end tests.
+
+CRI validation testing is currently Alpha and is hosted at the [cri-tools](https://github.com/kubernetes-incubator/cri-tools) repository. And performance benchmarking will be added in the future. We encourage the CRI developers to report bugs or help extend the test coverage by adding more tests.
+
+## Install
+
+The test suites can be installed easily via `go get` command:
+
+```sh
+go get github.com/kubernetes-incubator/cri-tools/cmd/critest
+```
+
+Then `critest` binary could be found in `$GOPATH/bin`.
+
+*Note: ensure GO is installed and GOPATH is set before installing critest.*
+
+## Running tests
+
+### Prerequisite
+
+Before running the test, you need to ensure that the CRI server under test is running and listening on a Unix socket. Because the validation tests are designed to request changes (e.g., create/delete) to the containers and verify that correct status is reported, it expects to be the only user of the CRI server. Please make sure that 1) there are no existing CRI-managed containers running on the node, and 2) no other processes (e.g., Kubelet) will interfere with the tests.
```

Comment:
Are you referring to a specific *test* when you say, "...ensure that the CRI server under test is running and listening..."?  If so, you might want to italicize it.